src/SME.SR.Application/Queries/BoletimEscolarDetalhado/MontarBoletinsDetalhados/MontarBoletinsDetalhadosQueryHandler.cs

```diff
@@ -67,7 +67,7 @@ public class
    
                 foreach (var aluno in alunos)
                 {
-                    var turma = turmas.First(t => aluno.Any(a => a.CodigoTurma.ToString() == t.Codigo));
+                    var turma = turmas.First(t => aluno.Any(a => t.TipoTurma == TipoTurma.Regular && (t.EtapaEnsino == 6 || t.EtapaEnsino == 9 || t.EtapaEnsino == 17) && a.CodigoTurma.ToString() == t.Codigo));
```

Comment:
Acho que aqui poderia criar constantes identificando cada valor de etapa, facilitando o entendimento.
E esse PR é pra HOM2, certo?

Reply:
Então Igor, minha sugestão seria pra deixar mais identificado ainda:
Por exemplo no topo da classe teriam variáveis constantes identificando cada tipo de etapa:
"private const int ENSINO_MEDIO = 6;
 private const int ENSINO_MEDIO_NORMAL = 9 . . . "
 Na base do EOL (se1426) é possível obter as identificações das etapas através dessa consulta:
 select * from etapa_ensino where cd_etapa_ensino in (6, 9, 17)